<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="sourceTreeEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="sourceTreeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="sourceTreeLabel">
     <property name="text">
      <string>Source tree:</string>
     </property>
     <property name="buddy">
      <cstring>sourceTreeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="dbButton">
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="diagramEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="diagramLabel">
     <property name="text">
      <string>Diagrams directory:</string>
     </property>
     <property name="buddy">
      <cstring>diagramEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="dbEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dbLabel">
     <property name="text">
      <string>Symbol database:</string>
     </property>
     <property name="buddy">
      <cstring>dbEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="diagramButton">
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceTreeEdit</tabstop>
  <tabstop>sourceTreeButton</tabstop>
  <tabstop>dbEdit</tabstop>
  <tabstop>dbButton</tabstop>
  <tabstop>diagramEdit</tabstop>
  <tabstop>diagramButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceTreeButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>selectSourceTree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>selectDatabase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diagramButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>selectDiagramsDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectSourceTree()</slot>
  <slot>selectDatabase()</slot>
  <slot>selectDiagramsDir()</slot>
 </slots>
</ui>
